Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:10.15.0-alpine as front
```

Comment:
I think you forgot to add those files to git:
- ./BuildApp.sh 
- .\BuildApp.bat